src/proto/FunctionRpc.proto

```diff
@@ -240,6 +240,13 @@ message FunctionEnvironmentReloadRequest {
 message FunctionEnvironmentReloadResponse {
   // Status of the response
   StatusResult result = 3;
+
+  // After specialization, worker sends capabilities & metadata.
+  // A map of worker supported features/capabilities
+  map<string, string> capabilities = 2;
```

Comment:
I added 2 and 1 for these new props as result was using 3. I looked at this history of this and found that PR #20 added this type and property and it never hard 1 and 2. So I am guessing this will not break anything. Let me know If I am wrong and should switch to 4 and 5

Reply:
I think one way to double check is - if you have these proto changes in Host/worker and if they are building properly then that's one validation it won't break. I have seen getting compiler errors if numbering is wrong.

Reply:
I don't understand protobuf enough to answer. But if you test a message with it and it works, good enough for me!

Reply:
nit - Just for readability, can we keep these fields in order?

Reply:
re-ordered the props. tested with host and compiles fine with generated classes.